plugins/htpasswd/package.json

```diff
@@ -34,29 +36,20 @@
     "unix-crypt-td-js": "1.0.0"
   },
   "devDependencies": {
-    "@commitlint/cli": "7.5.2",
-    "@commitlint/config-conventional": "7.5.0",
-    "@octokit/rest": "16.23.2",
-    "@types/bcryptjs": "2.4.2",
-    "@types/http-errors": "1.6.1",
-    "@types/jest": "24.0.11",
-    "@types/node": "11.12.1",
-    "@verdaccio/babel-preset": "0.1.0",
-    "@verdaccio/types": "5.0.0-beta.4",
-    "@verdaccio/eslint-config": "0.0.1",
-    "codecov": "3.2.0",
-    "cross-env": "5.2.0",
-    "eslint": "5.15.3",
-    "get-stdin": "6.0.0",
-    "husky": "0.14.3",
-    "in-publish": "2.0.0",
-    "jest": "24.5.0",
-    "lint-staged": "7.2.0",
-    "standard-version": "5.0.2",
-    "typescript": "3.2.4"
+    "@commitlint/config-conventional": "^7.5.0",
+    "@octokit/rest": "^16.23.2",
+    "@types/bcryptjs": "^2.4.2",
+    "@verdaccio/babel-preset": "^0.1.0",
+    "@verdaccio/types": "^5.0.0-beta.4",
+    "@verdaccio/eslint-config": "^0.0.1",
+    "cross-env": "^5.2.0",
+    "get-stdin": "^6.0.0",
+    "in-publish": "^2.0.0",
```

Comment:
Can you flatten the rest devDependencies to root package.json?

Reply:
not entirely sure what it means by flatten devDependencies, sorry guys

Reply:
I know this is complex to understand, we are improving CONTRIBUTING to add information about how to work with, create packages and importing packages. I will ask you to review if something is missing in this future CONTRIBUTING :wink:

With flatten devDependencies I mean to move devDependencies, that those code is not from this codebase, to the root package, so we can manage them in a single place and share between projects. 
By example, we cannot flatten `@verdaccio/eslint-config` because is a package in the codebase and **Yarn Workspaces** needs it defined in each workspace to link it correctly, but `in-publish` is a common devDependency that can be moved to the root `package.json` and manage one package version for every package that needs to use it.